src/main/java/com/exasol/adapter/dialects/exasol/ExasolColumnMetadataReader.java

```diff
@@ -52,4 +54,58 @@ public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
             return super.mapJdbcType(jdbcTypeDescription);
         }
     }
+
+    @Override
+    public JdbcTypeDescription readJdbcTypeDescription(final ResultSet remoteColumn) throws SQLException {
+        final JdbcTypeDescription typeDescription = super.readJdbcTypeDescription(remoteColumn);
+        if (typeDescription.getJdbcType() == EXASOL_INTERVAL_DAY_TO_SECONDS) {
+            return extractIntervalDayToSecondPrecision(remoteColumn, typeDescription);
+        } else if (typeDescription.getJdbcType() == EXASOL_INTERVAL_YEAR_TO_MONTHS) {
+            return extractIntervalYearToMonthPrecision(remoteColumn, typeDescription);
+        } else {
+            return typeDescription;
+        }
+    }
+
+    private String getTypeDescriptionStringForColumn(final ResultSet remoteColumn) throws SQLException {
+        try (final PreparedStatement preparedStatement = this.connection.prepareStatement(
+                "SELECT COLUMN_TYPE FROM SYS.EXA_ALL_COLUMNS WHERE COLUMN_SCHEMA = ? AND COLUMN_TABLE = ? AND COLUMN_NAME = ?;")) {
+            final String schema = remoteColumn.getString("TABLE_SCHEM");
```

Comment:
Is it really TABLE_SCHEM or is it a typo?

Reply:
Yes. https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html#getColumns(java.lang.String,%20java.lang.String,%20java.lang.String,%20java.lang.String)